test/paymentSplitter.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2018-present, Leap DAO (leapdao.org)
+ *
+ * This source code is licensed under the Mozilla Public License, version 2,
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+const PaymentSplitter = artifacts.require('PaymentSplitter');
+
+contract('PaymentSplitter', (accounts) => {
+
+  let paymentSplitter;
+
+  beforeEach(async () => {
+    paymentSplitter = await PaymentSplitter.new();
+  });
+
+  it('is splitable', async () => {
+    const balanceBefore = await web3.eth.getBalance(accounts[1]);
+    await paymentSplitter.split([accounts[1], accounts[2]], [500, 500], {value: '100000000'});
+    const balanceAfter = await web3.eth.getBalance(accounts[1]);
+    assert(balanceAfter !== balanceBefore, "split not performed");
```

Comment:
should we check the actually value transferred in tests?

Reply:
yeah, i didn't know how, will add.